### DISCUSSION TITLE
"Limitations page" should mentioning 1GB bandwidth, billing, etc.

### DISCUSSION THREAD

**Discussion by connorjclark** [git-lfs/git-lfs#5149] [General] [answered]
https://github.com/git-lfs/git-lfs/wiki/Limitations really should link to https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage .

**Comment by connorjclark** [comment]
I also suggest mentioning how Actions usage counts towards bandwidth, and that pretty much everyone should be using https://github.com/nschloe/action-cached-lfs-checkout to remediate that. big thanks to @nschloe <3

**Comment by bk2204** [comment, marked as answer]
Hey,

The wiki and this project are for the open source Git LFS client and not about GitHub, GitLab, or any other hosting platform.  GitHub's bandwidth limitations aren't a limitation of the Git LFS client, any more than a hosting platform's repository size limits are a limitation of Git.

As a result, it wouldn't really be appropriate to mention those there since they're not relevant to the open source project's limitations.

**Reply by connorjclark** [reply]
Hey, sorry about that. I didn't realize the distinction of https://git-lfs.github.com/ not being associated with GitHub, and assumed this was the right place for feedback to the GitHub team :/
